recipes/aicspylibczi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aicspylibczi" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+    sha256: d903caaac5576922f8dd4b1170fae3421fb9d203128b4d8ecb11681173e7046c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
This is running `cmake` behind the scenes, which relies on a vendored pybind11 submodule and some other stuff. We might need to patch things to make it conda-forge friendly.